### PR TITLE
Use "tighter" Jquery classes for extensions running on Index pages

### PR DESCRIPTION
**Pull request by WyohKnott**
It will prevent extensions from running on Reblog/Edit popups, and
*potentially* avoid creating conflicts with Editable reblogs.

### CODE REVIEW

**Comment by finagle29** [comment]
Nitpicking: when changing from the old versioning system to Semantic Versioning, I think the revisions should be updated into the patch number e.g. REV F => x.y.6.  Unless we want the patch numbering to only count patches that we've done.

File: Extensions/servant.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Servant **//
-//* VERSION 0.4 REV F **//
+//* VERSION 0.5 REV F **//
```
Comment (hobinjk) [inline Extensions/servant.js]: Nitpick: this would ideally be 0.4.7

File: Extensions/soft_refresh.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Soft Refresh **//
-//* VERSION 0.4 REV D **//
+//* VERSION 0.5 REV D **//
```
Comment (hobinjk) [inline Extensions/soft_refresh.js]: Nitpick: 0.4.5

File: Extensions/tagviewer.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 //* TITLE TagViewer **//
-//* VERSION 0.3 REV a **//
+//* VERSION 0.4 REV a **//
```
Comment (hobinjk) [inline Extensions/tagviewer.js]: Nitpick: 0.3.1

**Comment by hobinjk** [comment]
Manually tested for most of the extensions using #190. All of the changes follow the same basic structure so I'm not too worried about bugs. However, the ".posts .post" selector is in theory significantly slower than ".post". I've filed #192 to look into this and improving this general structure.

@homu r+

**Pull request merged**